File: Sem - 1 - C programms/fibonacci_series.c
// Roll no. - 12112010

#include <stdio.h>
int main()
{
    int n, t1 = 0, t2 = 1, nextTerm = 0, i;
    printf("Enter the n value: ");
    scanf("%d", &n);

    if (n == 0 || n == 1)
        printf("%d", n);
    else
        nextTerm = t1 + t2;

    for (i = 3; i <= n; ++i)
    {
        t1 = t2;
        t2 = nextTerm;
        nextTerm = t1 + t2;
    }

    printf("%d",nextTerm);

    return 0;
}

// {

//     int n, curr = 1, prev = 1, next;
//     printf("Enter the number: ");
//     scanf("%d", &n);

//     for (int i = 1; i <= n; i++)
//     {
//         if (i == 1 || i == 2)
//         {
//             printf("%d ", curr);
//             if (i == 2)
//                 curr++;
//         }
//         else
//         {
//             printf("%d ", curr);
//             next = curr + prev;
//             prev = curr;
//             curr = next;
//         }
//     }
//     return 0;
// }

File: Pyhon codes/Sem - 2 Python Codes/fibonacci series.py
# Name: Ashish Kumar Yadav
# Roll no.: 12112010
# Branch: I.T

# Write a Program to print Fibonacci series upto n terms.

n = int(input("Enter the value of n:"))
current = 0
next = 1
prev = 0
print("The Fibonacci series upto", n, "terms is:")
print(0)
print(1)
for i in range(1, n - 1, 1):
    current = next+prev
    print(current)
    prev = next
    next = current
